### PR TITLE
Feature/9144

### PR DESCRIPTION
**Pull request by accbjt**

### CODE REVIEW

**Comment by accbjt** [comment]
@jbalboni Let me know if I put my css file in the correct place. I did not see any examples of where we put css code for specific elements.

**Comment by jbalboni** [comment]
@accbjt Can you include a screenshot of your change in here? We try to do that for most PRs.

**Comment by accbjt** [comment]
@jbalboni here is a screenshot of what it should look like.
screen shot 2018-05-07 at 12 10 08 pm

File: src/platform/site-wide/sass/schedule-an-appointment.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+button#crisis-expander-link {
```
Comment (jbalboni) [inline src/platform/site-wide/sass/schedule-an-appointment.scss]: We have a `va-button-link` class that should turn a button into a correctly styled link. Will that work for your use case?

Comment (accbjt) [reply src/platform/site-wide/sass/schedule-an-appointment.scss]: Okay I will try using it.

**Review by accbjt — COMMENTED** [review]
Made the change

**Review by jbalboni — APPROVED** [review]

**Pull request merged**